app/views/layouts/application.html.erb

```diff
@@ -18,24 +18,36 @@
     <%= stylesheet_pack_tag 'application', media: 'all' %>
     <%= javascript_pack_tag 'application', defer: true unless params[:nojs] === "nojs" %>
 
+    <!-- Google Tag Manager (doesn't store personal info until permission given) -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-N58Z5PG');</script>
+    <!-- End Google Tag Manager -->
+
     <% if cookies[:cookie_consent_1] == "on" %>
-      <script async src="https://www.googletagmanager.com/gtag/js?id=G-LR5YHTJD65"></script>
+      <script async src="https://www.googletagmanager.com/gtag/js?id=G-5707QPHZE0"></script>
     <% else %>
       <script>
         // For adding the script once the user has given permission
-        window.ecfAnalyticsScript = "https://www.googletagmanager.com/gtag/js?id=G-LR5YHTJD65";
+        window.ecfAnalyticsScript = "https://www.googletagmanager.com/gtag/js?id=G-5707QPHZE0";
       </script>
     <% end %>
```

Comment:
I think this value should only be used in production?

Reply:
We also seem to be missing the second script?

Reply:
have changed it. i don't think it's worth putting in an environmental variable but if anyone else does i'm happy to change it